### ISSUE TITLE
Revise documentation of `<source>`

### ISSUE THREAD

**Issue by bwbohl** [music-encoding/music-encoding#1496]
In v4.0.0 `<source>` underwent heavy changes that are not reflected in the documentation. This leads to confusion (cf. https://github.com/music-encoding/music-encoding/issues/579) and should be fixed

**Comment by bwbohl** [comment]
connected to music-encoding/guidelines#82

**Comment by musicEnfanthen** [comment]
@riedde @doerners @bwbohl May I ask you to have a look if this is still an ongoing issue? (At least, all corresponding issues that I could find on music-encoding/music-encoding have been closed.)

**Comment by doerners** [comment]
I just had a quick look at the element definition of `<source>` and the guidelines as well. It seems that `<source>` is sufficiently documented. There are also several notes pointing out that in previous MEI versions `<source>` was used for manifestation-level descriptions - which schould aid to alienate the confusion. So to me it seems that this issue could be closed if @bwbohl and @riedde don't object.

**Comment by riedde** [comment]
yes, @doerners is right. In the guidelines (3.2) it is clarified. But in 3.4.1.6 an additional sentence (as in 3.5.1 'Please note: Until MEI 3.0.0, the source element in sourceDesc was used for manifestation-level descriptions.') would be nice.
In 3.8.3 the example clarifies the distinction.

in the End 12.1.1 has to be improved, because this is really unclear or outdated (focussing on the source element).

**Comment by musicEnfanthen** [comment]
Transferring to main repo

**Comment by bwbohl** [comment]
closed with db53d16d1d639d80a00173d28346a10de3684f4a
